sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStore.java

```diff
@@ -202,6 +193,7 @@ public void storeStatus(Protos.TaskStatus status) throws StateStoreException {
         } catch (PersisterException e) {
             throw new StateStoreException(e);
         }
+        return currentStatusOptional;
```

Comment:
The naming of `currentStatusOptional` is a little misleading now that we return it. The comments also mention the "previous `TaskStatus`".